webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.js

```diff
@@ -17,7 +17,7 @@ const PageLayout = ({
   children,
 }) => (
   <div id="main">
-    <div id="content">
+    <div id="react-content">
```

Comment:
This may break several pages which rely the `#content` selector once they use this layout - some things that come to mind are full screen editor and host form submit (search for `#content` to find all references)

Reply:
do we still need these functionality in a pure react page?

Reply:
@tbrisker I agree with @amirfefer.  Views that use PageLayout should not rely on the legacy selector.

Reply:
@tbrisker yeah I would agree, if full react pages is what we want, we should not have legacy stuff taking effect there.